app/templates/components/events/view/overview/event-tickets.hbs

```diff
@@ -15,7 +15,7 @@
       </tr>
       <tr>
         <td>{{t 'Sales (amount)'}}</td>
-        <td>US$ {{number-format data.orderStat.sales.completed}}</td>
+        <td>{{ data.event.paymentCurrency}} {{number-format data.orderStat.sales.completed}}</td>
```

Comment:
why are you not using the helper here?

Reply:
As i was looking into 'event-tickets.hbs' none of the data attribute was using a helper and introducing a helper in this file would need to modify the tests to make it pass the travis .

Reply:
so, you opened this PR to parse the correct currency. Currency is `parsed` using the helper. And that obviously requires modifying the component test. So you decide to remove the parser itself instead of fixing the test, making the premise of your PR redundant? Tests are written by the developers and run using Travis, so when you change something in a component, and it happens to have a test associated with it, you modify the test, not abandon the idea of making the change.

> none of the data attributes was using a helper

All the other places where you have used the helper are identical data attributes, some were passed directly some in a nested form and hence written as data.event.paymentCurrency instead of paymentCurrency. There is no actual difference between them other than the semantically different references required to access them.